doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,286 @@
+
+The transition to other initial states is done by rotating the whole system, and the rotation is either explicitly specified or computed to rotate one vector to another.
+Let :math:`v_1, ..., v_K \in \mathbb{R}^3` be the vectors defining the initial configuration of a 3d geometry, and :math:`v_1^{(\text{def})}, ..., v_K^{(\text{def})}` are their default values.
+The rotated configuration is given by
+
+    .. math::
+        v_k = R^{(\mathrm{i})}\, v_k^{(\text{def})},
+
+i.e., *all vectors are transformed by the same rotation*.
+The matrix :math:`R^{(\mathrm{i})} \in \mathbb{R}^{3 \times 3}` is chosen to rotate the first vector from its default :math:`v_1^{(\text{def})}` to its actual value :math:`v_1`, i.e.,
+
+    .. math::
+        R^{(\mathrm{i})} v_1^{(\text{def})} = v_1.
+
+Since the rotation :math:`R^{(\mathrm{i})}` is not uniquely determined, we choose to perform a rotation in the plane spanned by :math:`v_1^{(\text{def})}` and :math:`v_1`, making use of `Rodrigues' rotation formula <https://en.wikipedia.org/wiki/Rodrigues%27_rotation_formula>`_:
+
+    .. math::
+        a &= \frac{v_1^{(\text{def})} \times v_1}{\big| v_1^{(\text{def})} \times v_1 \big|},\ \cos\beta = \langle v_1^{(\text{def})},\, v_1 \rangle,
+
+        R^{(\mathrm{i})} v &= \cos\beta v + \sin\beta (a \times v) + (1 - \cos\beta)\langle a,\, v \rangle a.
+
+This construction becomes unstable when :math:`v_1^{(\text{def})} \approx v_1`.
+Therefore, if :math:`\big\| v_1^{(\text{def})} - v_1 \big\| < \delta` for a threshold :math:`\delta`, we take :math:`v_1 = v_1^{(\text{def})}`.
+
+Below are two concrete examples for how this computation is done in practice.
+For a list of implemented geometries, check the API documentation of the ``odl.tomo.geometry`` subpackage.
+
+`Parallel3dEulerGeometry`
+^^^^^^^^^^^^^^^^^^^^^^^^^
+In this geometry, the motion parameters are two or three Euler angles, and the detector is two-dimensional and flat.
+The handle :math:`v_1` for the initial rotation is the initial detector position, provided as ``det_pos_init`` parameter.
+Its default value is :math:`v_1^{(\text{def})} = (0, 1, 0)^{\mathrm{T}}`.
+
+`Parallel3dAxisGeometry`
+^^^^^^^^^^^^^^^^^^^^^^^^
+This geometry, like all subclasses of `AxisOrientedGeometry`, has a fixed symmetry axis, provided as ``axis`` parameter.
+Its motion parameter is a single rotation angle around this axis.
+The initial orientation handle :math:`v_1` is the symmetry axis, with default value :math:`v_1^{(\text{def})} = (0, 0, 1)^{\mathrm{T}}`.
+
+`CircularConeFlatGeometry`
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+The 3D cone beam geometry with circular acquisition curve is also an `AxisOrientedGeometry`.
+Here, the symmetry ``axis`` is perpendicular to the source and detector circles (which can be different but lie in the same plane).
+Its motion parameter is a single angle that parametrizes the position of the source on the circle, and the detector lies opposite of the source point, at maximum distance.
```

Comment:
"at maximum distance" perhaps just remove this since it does not add much and is somewhat confusing. "opposite" is enough.